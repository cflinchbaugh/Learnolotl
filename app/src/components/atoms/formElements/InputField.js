import React from 'react';
import styled from 'styled-components';

const StyleWrapper = styled.div`
    
`

function InputField (props) {
    return (
        <StyleWrapper>
            <div>{props.label}</div>
            <input {...props}></input>
        </StyleWrapper>
    )
}

export default InputField