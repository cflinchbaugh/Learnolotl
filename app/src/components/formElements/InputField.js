import React from 'react';
import styled from 'styled-components';

const StyleWrapper = styled.div`
    input {
        background-color: #535d81;
        color: white;
        border: solid 1px white;
        margin-bottom: 25px;
    }
`

function InputField (props) {
    return (
        <StyleWrapper>
            <div>{props.label}</div>
            <input {...props}></input>
        </StyleWrapper>
    )
}

export default InputField