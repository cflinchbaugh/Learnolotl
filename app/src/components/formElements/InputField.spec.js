import React from 'react';
import {shallow} from 'enzyme';
import InputField from './InputField';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('InputField', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const inputFieldData = {
        }

        const tree = renderer.render(
            <InputField {...inputFieldData}/>
        );

        expect(tree).toMatchSnapshot();
    });

});