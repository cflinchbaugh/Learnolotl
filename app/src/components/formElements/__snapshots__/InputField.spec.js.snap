// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputField renders and matches snapshot 1`] = `
<styled.div>
  <div />
  <input />
</styled.div>
`;
